setup_lacework_agent.sh

```diff
@@ -39,4 +40,11 @@ cat >"$LACEWORK_INSTALL_PATH/config/config.json" <<EOF
 }
 EOF
 
+# Make sure the Lacework datacollector service is enabled and running
+if ! systemctl is-active --quiet datacollector; then
+  echo "Enabling the Lacework datacollector service"
+  systemctl enable datacollector
```

Comment:
what if the service is enabled but not active? would this fail? 🤔

Reply:
<img width="1059" alt="Screen Shot 2021-03-02 at 2 17 20 PM" src="https://user-images.githubusercontent.com/3448802/109702286-07a1fc80-7b62-11eb-85ab-924dca19681f.png">

I just did the test, I confirm I confirm enabling an already enabled service does not cause any issue.